lib/gphoto2/camera_abilities.rb

```diff
@@ -32,8 +32,11 @@ def [](field)
 
     # @return [Integer]
     def operations
-      return 0 if self[:operations] === :none
-      self[:operations]
+      if self[:operations] === :none
```

Comment:
Use `==` here (see [style guide](https://github.com/bbatsov/ruby-style-guide#no-case-equality)), and this patch should be good to go.